docker-cpp-demos/rtsp-docker/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM ubuntu:18.04
+
+RUN apt-get upgrade
+RUN apt-get update
+
+RUN apt-get install -y git && \
```

Comment:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments

Reply:
Fixed

Reply:
I think the packages can be installed with a single `apt-get install`